backend/src/main/java/ca/bc/gov/app/routes/ches/ChesRouter.java

```diff
@@ -0,0 +1,95 @@
+package ca.bc.gov.app.routes.ches;
+
+import static org.springdoc.core.fn.builders.apiresponse.Builder.responseBuilder;
+import static org.springdoc.core.fn.builders.content.Builder.contentBuilder;
+import static org.springdoc.core.fn.builders.exampleobject.Builder.exampleOjectBuilder;
+import static org.springdoc.core.fn.builders.header.Builder.headerBuilder;
+import static org.springdoc.core.fn.builders.requestbody.Builder.requestBodyBuilder;
+import static org.springdoc.core.fn.builders.schema.Builder.schemaBuilder;
+import static org.springdoc.webflux.core.fn.SpringdocRouteBuilder.route;
+import static org.springframework.web.reactive.function.server.RequestPredicates.accept;
+
+import ca.bc.gov.app.dto.ches.ChesRequest;
+import ca.bc.gov.app.handlers.ches.ChesHandler;
+import ca.bc.gov.app.routes.BaseRouter;
+import java.util.function.Consumer;
+import lombok.RequiredArgsConstructor;
+import org.apache.commons.lang3.StringUtils;
+import org.springdoc.core.fn.builders.operation.Builder;
+import org.springframework.http.MediaType;
+import org.springframework.stereotype.Component;
+import org.springframework.web.reactive.function.server.RouterFunction;
+import org.springframework.web.reactive.function.server.ServerResponse;
+
+@Component
+@RequiredArgsConstructor
+public class ChesRouter extends BaseRouter {
+
+  private final ChesHandler handler;
+
+  @Override
+  public String basePath() {
+    return "/api/mail";
```

Comment:
can we move hardcoded strings to a constant file?

Reply:
We can, but what will be the benefit? This specific string is being used only here. At the end the compiler would inject the value here anyway after compiling.

Reply:
I believe I made the statement confusing, I was coming more from the angle of basepath which would be reused across different routers for example: "api/v1", "api/v2" et..

Reply:
Oh gotcha. Sure, will move the "/api" part to the base router and keep the rest in here. Thanks